app/models/patient.rb

```diff
@@ -40,6 +40,10 @@ def unblock!
     update!(login_attempts: 0)
   end
 
+  def years_old
+    (Date.now.to_date - Date.parse(patient.birth_date)).to_i / 365
```

Comment:
This line broke the application, after a few tests, ran correctly changing to:

`(Date.today.to_date - Date.parse(birth_date)).to_i / 365`

This logic concede to the pacient 15 days "*de lambuja*", if consider each extra day in leap years.. By me, it's OK :+1: